docs/operator-guide/cd/auto-stable-trigger-type.md

```diff
@@ -0,0 +1,59 @@
+# Deployment Strategies Overview
+
+Deploying applications to production environments involves risks of downtime and service disruptions. To mitigate such issues and enhance stability, KubeRocketCI provides several deployment strategies.
+
+## Deployment Strategies in KubeRocketCI
+
+In KubeRocketCI application deployment strategies are managed by the trigger type resource. Application deployment can be triggered once the new application version is released. This behavior is defined by trigger types. Trigger type is a KubeRocketCI-specific resource that defines how Environment will behave once new application version is built. There are three trigger types:
+
+* **Manual**: Requires a user to manually initiate the deployment process after a new application version is ready.
+* **Autodeploy**: Automatically triggers the deployment pipeline as soon as a new application version is built, ensuring the latest version is deployed.
```

Comment:
latest version is deployed - please change. Here we need to provide information that all application was deployed with the latest version